src/main/java/carpet/script/CarpetExpression.java

```diff
@@ -822,11 +820,7 @@ else if (arg instanceof EntityValue)
      * <p>Triple of entity motion vector, <code>l(motion_x, motion_y, motion_z)</code></p>
      * <h3><code>query(e,'motion_x'), query(e,'motion_y'), query(e,'motion_z')</code></h3>
      * <p>Respective component of the motion vector</p>
-     * <h3><code>query(e,'name')</code></h3>
-     * <p>String of entity name</p>
-     * <h3><code>query(e,'custom_name')</code></h3>
-     * <p>String of entity name</p>
-     * <h3><code>query(e,'type')</code></h3>
+     * <h3><code>query(e,'name'), query(e,'custom_name'), query(e,'type')</code></h3>
```

Comment:
Thanks a lot for spotting that. Totally missed it. They are actually three different features. Example (Villager):
e ~ 'name' => 'Weapon Smith'
e ~ 'custom_name' => null
e ~ 'type' => 'villager'
If you could fix that in the docs, that would be great

Reply:
Actually, don't worry. I can change it myself in github